apps/coscrad-frontend/src/store/slices/noteSlice/thunks/fetchNotes.ts

```diff
@@ -0,0 +1,8 @@
+import { getConfig } from '../../../../config';
+import { createFetchThunk } from '../../utils/createFetchThunk';
+import { NOTES } from '../constants';
+
+export const fetchNotes = createFetchThunk(
+    `${NOTES}/fetch`,
```

Comment:
We probably want to share this entire string wherever it is used. Keep in mind how we might manage these action type prefixes / slice identifiers / sub-reducer names.